onnxruntime/test/onnx/TestCase.cc

```diff
@@ -203,7 +204,12 @@ class OnnxModelInfo : public TestModelInfo {
     if (!model_pb.ParseFromZeroCopyStream(&f)) {
       ORT_THROW("Failed to load model because protobuf parsing failed.");
     }
-
+#ifdef __GNUG__
+    std::string url_string{model_url};
+    if (url_string.size() >= 18) {
+      onnx_commit_tag_ = {url_string.begin()+11, url_string.begin()+18};  
```

Comment:
this seems a bit brittle. is there a better way?

Reply:
Two suggestions:
1. using regex;
2. split path by '/', then take an elem from the string array.
Ideas?